cli/dev/image.go

```diff
@@ -16,7 +16,7 @@ const DEFAULT_BASE_OS_VALUE = "alpine:latest"
 const MAKE_FILE_CONTENT = `# Make targets for building the IBM example helloworld edge service
 
 # This imports the variables from horizon/hzn.cfg. You can ignore these lines, but do not remove them.
--include horizon/.hzn.cfg.tmp.mk
+-include PROJECT_DIR/.hzn.cfg.tmp.mk
```

Comment:
Also a nit, but the file should now be called `.hzn.json.tmp.mk`

Reply:
done